tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -268,6 +269,8 @@ Copyright (C) Aqua Security inc.
 #define MAX_BIN_CHUNKS                  110
 #endif
 
+#define IOCTL_SYSCALL_HOOK              65
```

Comment:
Any reason for using 65?

Reply:
nope , it just a random number because it just for verification.
I can added a comment about it